packages/api/src/routes/search.routes.ts

```
@@ -0,0 +1,760 @@
+
+    // TODO: Coordinate with Agent 5's export service
+    // For now, return a simple JSON export
+    let exportData: string;
+
+    switch (format) {
+      case 'json':
+        exportData = JSON.stringify(results, null, 2);
+        break;
+      case 'csv':
+        // Simple CSV export
+        const headers = ['Title', 'Authors', 'Year', 'DOI', 'Citation Count'];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)